swo_mpt_api/httpquery.py

```diff
@@ -0,0 +1,68 @@
+from typing import Annotated, Generic, Self, TypeVar
+
+from mpt_extension_sdk.mpt_http.base import MPTClient
+
+T = TypeVar("T")
+
+
+class HttpQuery(Generic[T]):
```

Comment:
mmm... could you talk to @svazquezco  also, I think we already have some interface and implementation in CLI tool... What do you think about to extract it to the separate library? Maybe it is a good timing now?